lessons/linear-regression/langs/en-us/pages/drawing-predictors.adoc

```diff
@@ -10,11 +10,22 @@ td { margin: 0; padding: 0; }
 
 For each of the scatter plots below, draw a *predictor line* that seems like the best fit. Describe the correlation in terms of Direction and Strength, then estimate the @math{r}-value as being close to -1, -0.5, 0, +0.5, or +1.
 
+@vspace{1ex}
+
+*Remember what we learned about r-values...*
```

Comment:
Can we move this table to the top of the page? The directions make more sense coming after the review